Makefile

```diff
@@ -237,13 +237,13 @@ test-sim:
 	@VERSION=$(VERSION) go test -mod=readonly $(PACKAGES_SIM)
 
 test-e2e:
-	@VERSION=$(VERSION) go test -mod=readonly -timeout=25m -v $(PACKAGES_E2E)
+	@VERSION=$(VERSION) go test -mod=readonly -timeout=25m -v github.com/osmosis-labs/osmosis/v7/tests/e2e
```

Comment:
so we're no longer testing the current code base?

Reply:
As per our discussion, will revert